main.py

```diff
@@ -126,9 +126,15 @@ async def on_message(self, message) -> None:
 
                 if message.channel.id not in command_function.command_data["allowed_channels"] \
                         and message.channel.id not in configuration.ALLOWED_COMMAND_CHANNELS:
-
+                    
+                util.choose_random
+               
                     await message.channel.send(f"Please use <#{configuration.DEFAULT_COMMAND_CHANNEL}> for bot commands!")
                     return
+                or
```

Comment:
Invalid syntax?
Should be indented and the new-line should be escaped in line 128